taichi/program/kernel_profiler.h

```diff
@@ -12,26 +12,34 @@
 
 TLANG_NAMESPACE_BEGIN
 
-struct KernelProfileRecord {
+struct KernelProfileTracedRecord {
+  std::string name;
+  float kernel_elapsed_time_in_ms{0.0};
```

Comment:
TODO: In the future we can use a more accurate type to denote time and duration, e.g. `std::chrono::duration`